flutter/lib/src/native/c/binding.dart

```diff
@@ -869,7 +869,7 @@ class SentryNative {
 /// frozen but lists and dicts are not and can be frozen on demand.  This
 /// automatically happens for some shared values in the event payload like
 /// the module list.
-class sentry_value_u extends ffi.Union {
+base class sentry_value_u extends ffi.Union {
```

Comment:
these should be `final class`, see https://github.com/getsentry/sentry-dart/blob/20926fca22a75ad0f1aaaebc7693f5d17570674c/flutter/scripts/generate-native-bindings.ps1#L13

Reply:
Also can you please update the script?

Reply:
Am I correct in the assumption that we'd want to remove those lines?

Reply:
yes, that's correct